components/ILIAS/Data/src/TextHandling/Shape/WordOnlyMarkdown.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ *
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ *********************************************************************/
+
+declare(strict_types=1);
+
+namespace ILIAS\Data\TextHandling\Shape;
+
+use ILIAS\Data\TextHandling\Structure;
+use ILIAS\Data\TextHandling\Text;
+
+class WordOnlyMarkdown extends SimpleDocumentMarkdown
+{
+    /**
+     * @return mixed[] consts from Structure
```

Comment:
```suggestion
     * @return Structure[]
```

or simply remove the docs because of the definition in the paren class.